pkg/bootstrap/docker/openshift/login.go

```diff
@@ -32,6 +33,32 @@ func Login(username, password, server, configDir string, f *clientcmd.Factory, c
 	for k := range adminConfig.AuthInfos {
 		adminConfig.AuthInfos[k].LocationOfOrigin = ""
 	}
+	serverFound := false
+	for k := range adminConfig.Clusters {
+		adminConfig.Clusters[k].LocationOfOrigin = ""
+		if adminConfig.Clusters[k].Server == server {
+			serverFound = true
```

Comment:
Did you miss `break` here or we're have to assign `LocationOfOrigin` to an empty string for all clusters?